_playwright-tests/UI/helpers/helpers.ts

```diff
@@ -14,3 +14,45 @@ export const closePopupsIfExist = async (page: Page) => {
     });
   }
 };
+export const filterByName = async (page: Page, name: string) => {
+  await page.getByPlaceholder(/^Filter by name.*$/).fill(name);
+};
+
+export const clearFilters = async (page: Page) => {
+  try {
+    await page.getByRole('button', { name: 'Clear filters' }).waitFor({ timeout: 5000 });
+  } catch {
+    return Promise<void>;
```

Comment:
This seems like a strange return type. 
Would return; not work here?

Reply:
it probably would, this just explicitly specifies the type instead of it being inferred, but I can change that if desired 😅

Reply:
changed 🫡